dev-docs/bidders/dspx.md

```diff
@@ -20,13 +23,13 @@ The DSPx Bidder Adapter requires setup and approval from DSPx team. Please reach
 | Name          | Scope    | Description                                                                | Example                | Type            |
 |---------------|----------|----------------------------------------------------------------------------|------------------------|-----------------|
 | `placement`   | required | Placement ID from dspx.                                                    | `'101'`                  | `string`        |
-| `pfilter`     | optional | Selection filter. E.g. floorprice (min_cpm_micros)                            | `{"floorprice": 1000000}`| `object`        |
+| `pfilter`     | optional | Custom filter parameters.                            | `{"customxy": 1000000}`| `object`        |
```

Comment:
There are no references to `customxy` in your corresponding adapter PR changes, I still see `floorprice` referenced from `pfilter`?

Reply:
`customxy` is an example. we share the actual custom filter parameters with partners on-demand when adding them to their account.

the old optional pfilter option `floorprice` is deprecated, no longer required if the inventory partner is using the pbjs-internal floorprice. if you don't mind, for deprecation and temporary downw.-compatibility for old integrations, we'd keep it in the code temporarily but remove from docs already by this update.